include/firefly/intel64/cpu/cpu.hpp

```diff
@@ -2,6 +2,7 @@
 
 #include "firefly/intel64/gdt/gdt.hpp"
 #include "firefly/intel64/gdt/tss.hpp"
+#include <libk++/bits.h>
```

Comment:
Again, we have no explicit coding guidelines yet but this would be in double qoutes.
Only cxxshim and frigg includes are `<>`